File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { StitchAiDesignSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await StitchAiDesignSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- StitchAiDesign SDK exists test

local sdk = require("stitch-ai-design_sdk")

describe("StitchAiDesignSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
